### ISSUE TITLE
is posible run i18n-tasks add-missing excluding keys ?

### ISSUE THREAD

**Issue by SamZapata** [glebm/i18n-tasks#323]
Hi. I'm using the command 'add-missing' for keys generation but this also add the default keys like 'date.abbr_month_names'. Can these be ignored?

**Comment by espen** [comment]
Yes see https://github.com/glebm/i18n-tasks/blob/master/templates/config/i18n-tasks.yml#L95

**Comment by davidwessman** [comment]
The solution here is to configure `ignored` keys
